ansible/roles/dashd/tasks/main.yml

```diff
@@ -103,12 +103,16 @@
       wallet={{ wallet_rpc_wallet_mno }}
   when: enable_wallet is true and wallet_exists.stat.exists is true
 
-# TODO: why does this always take exactly 30 seconds on first deploy?
+- name: Stop dash core
+  community.docker.docker_compose:
+    project_src: '{{ dashd_compose_path }}'
+    state: absent
```

Comment:
Shouldn't this be
```suggestion
    state: present
    stopped: true
```

Reply:
No this is a step, it's absent. The following lines make it present again